mithril-aggregator/src/database/provider/signed_entity.rs

```diff
@@ -63,6 +68,63 @@ where
     }
 }
 
+impl TryFrom<SignedEntityRecord> for SnapshotMessage {
+    type Error = StdError;
+
+    fn try_from(value: SignedEntityRecord) -> Result<Self, Self::Error> {
+        #[derive(Deserialize)]
+        struct TmpSnapshot {
```

Comment:
Why not call it `SnapshotArtifact`. Unless it's important to point out that it's temporary.

Reply:
I took the same name as the artifact it replaces `Snapshot` and added the temporary prefix.